2-library/unit-test-bundle/src/main/java/com/_6core/lib/unit/test/bundle/package-info.java

```diff
@@ -0,0 +1,8 @@
+/**
+ * JUnit and Mockito have many dependencies that take up
+ * a lot of space and create a messy dependency block in pom.xml
+ * This module contains one library that will have everything
+ * these dependencies to just inject just one
+ */
```

Comment:
if you want to leave a comment in package-info you need to describe the needs of the package
this comment describes likely the whole library, not the package

for libs description you can use `README.md`
```suggestion
```

Reply:
The idea was not well thought out. Just deleted it. )